src/lib/__tests__/utils.test.ts

```diff
@@ -54,4 +54,43 @@ describe('addCustomMODUKClass', () => {
       'matching',
     )
   })
+
```

Comment:
Here, I feel there a a lot of repetition, what do you think about adding fixtures in `describe.each`

Reply:
I'll do this in another PR